src/redismodule.h

```diff
@@ -348,7 +355,15 @@ static const RedisModuleEvent
     RedisModuleEvent_ForkChild = {
         REDISMODULE_EVENT_FORK_CHILD,
         1
-    };
+    },
+    RedisModuleEvent_BeforeSleep = {
+        REDISMODULE_EVENT_BEFORE_SLEEP,
+        1
+    },
+    RedisModuleEvent_AfterSleep = {
+        REDISMODULE_EVENT_AFTER_SLEEP,
+        1
```

Comment:
these two must be documented in `RM_SubscribeToServerEvent`.
p.s. maybe they should be one event with two sub-events?

if we do that, and even if we don't, we need to add their entries to RM_IsSubEventSupported

Reply:
OK, made it one event with two sub-event and added to the RM_IsSubEventSupported